pkg/hub/deployer/generic/generic.go

```diff
@@ -61,6 +62,23 @@ type Deployer struct {
 	apiserverURL string
 }
 
+func (deployer *Deployer) Enqueue(obj runtime.Object) {
```

Comment:
This would be redundant if all can be done on agent side.

Reply:
Yes, I keep hub deployer not changed.